frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1093,320 +1095,588 @@ function ReaderFooter:addToMainMenu(menu_items)
+                callback = function()
+                    self.settings.toc_markers = not self.settings.toc_markers
+                    self:setTocMarkers()
+                    self:refreshFooter(true)
+                end
+            },
+            {
+                text_func = function()
+                    local markers_width_text = _("thick")
+                    if self.settings.toc_markers_width == 1 then
+                        markers_width_text = _("thin")
+                    elseif self.settings.toc_markers_width == 2 then
+                        markers_width_text = _("medium")
+                    end
+                    return T(_("Chapter marker width: %1"), markers_width_text)
+                end,
+                enabled_func = function()
+                    return not self.settings.progress_style_thin and not self.settings.chapter_progress_bar
+                        and self.settings.toc_markers and not self.settings.disable_progress_bar
+                end,
+                sub_item_table = {
+                    {
+                        text = _("Thin"),
+                        checked_func = function()
+                            return self.settings.toc_markers_width == 1
+                        end,
+                        callback = function()
+                            self.settings.toc_markers_width = 1  -- unscaled_size_check: ignore
+                            self:setTocMarkers()
                             self:refreshFooter(true)
                         end,
                     },
                     {
-                        text = _("Left"),
+                        text = _("Medium"),
                         checked_func = function()
-                            return self.settings.align == "left"
+                            return self.settings.toc_markers_width == 2
                         end,
                         callback = function()
-                            self.settings.align = "left"
+                            self.settings.toc_markers_width = 2  -- unscaled_size_check: ignore
+                            self:setTocMarkers()
+                            self:refreshFooter(true)
+                        end,
+                    },
+                    {
+                        text = _("Thick"),
+                        checked_func = function()
+                            return self.settings.toc_markers_width == 3
+                        end,
+                        callback = function()
+                            self.settings.toc_markers_width = 3  -- unscaled_size_check: ignore
+                            self:setTocMarkers()
                             self:refreshFooter(true)
+                        end
+                    },
+                },
+            },
+        }
+    })
+    ---------------------------------
+    -- footer_items
+    local footer_items = {}
+    table.insert(sub_items, {
+        text = _("Items"),
```

Comment:
```suggestion
        text = _("Status bar items"),
```
?

Reply:
Seems fine (but I guess that was for @poire-z )